src/modules/Builder/hooks/useActivityFlowsRedirection.ts

```diff
@@ -1,25 +1,29 @@
 import { useEffect } from 'react';
 import { useParams, useNavigate, generatePath } from 'react-router-dom';
 import { useFormContext } from 'react-hook-form';
-import debounce from 'lodash.debounce';
 
 import { page } from 'resources';
-import { applet } from 'shared/state';
+import { useCheckIfNewApplet } from 'shared/hooks';
```

Comment:
useCheckIfNewApplet  used just in Builder/

Reply:
it's being used in Dashboard too (dashboard - applet - settings - report config)